### ISSUE TITLE
Custom clock format

### ISSUE THREAD

**Issue by fitrh** [MarianArlt/kde-plasmoid-chiliclock#14]
Is there any way to custom the clock format like how we custom the date format ? Because i want to add leading zero for 12h format but i can't find how to do that, if i put `hh` on Date Format, it will display `hh` instead of 12h with leading zero

**Comment by mrgatesjunior** [comment]
Yes, also please provide an option to turn on and off the clock as well.  This way I could put the widget in the middle of the dock to just show date info and then another on the far right on the dock to just display time.

It is really near perfect minus a couple of the above suggestions.

**Comment by mndscp** [comment]
Yes, leading zero please! That would be awesome. Thanks in advance :)
